Resources/public/js/cmf_tree_browser.fancytree.js

```diff
@@ -1,2 +1,5930 @@
+
+/******/ 		// Create a new module (and put it into the cache)
+/******/ 		var module = installedModules[moduleId] = {
+/******/ 			exports: {},
+/******/ 			id: moduleId,
+/******/ 			loaded: false
+/******/ 		};
+
+/******/ 		// Execute the module function
+/******/ 		modules[moduleId].call(module.exports, module, module.exports, __webpack_require__);
+
+/******/ 		// Flag the module as loaded
+/******/ 		module.loaded = true;
+
+/******/ 		// Return the exports of the module
+/******/ 		return module.exports;
+/******/ 	}
+
+
+/******/ 	// expose the modules object (__webpack_modules__)
+/******/ 	__webpack_require__.m = modules;
+
+/******/ 	// expose the module cache
+/******/ 	__webpack_require__.c = installedModules;
+
+/******/ 	// __webpack_public_path__
+/******/ 	__webpack_require__.p = "";
+
+/******/ 	// Load entry module and return exports
+/******/ 	return __webpack_require__(0);
+/******/ })
+/************************************************************************/
+/******/ ([
+/* 0 */
+/***/ function(module, exports, __webpack_require__) {
+
+	'use strict';
+
+	var _jquery = __webpack_require__(1);
+
+	var _jquery2 = _interopRequireDefault(_jquery);
+
+	var _fancytree = __webpack_require__(2);
+
+	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+	/**
+	 * A simple layer between the jQuery front-end and the JS tree library used.
+	 *
+	 * By default, it uses the FancytreeAdapter. You can pass other adapters by
+	 * changing the `adapter` setting.
+	 *
+	 * @author Wouter J <wouter@wouterj.nl>
+	 */
+	/*
+	 * This file is part of the Symfony CMF package.
+	 *
+	 * (c) 2011-2017 Symfony CMF
+	 *
+	 * For the full copyright and license information, please view the LICENSE
+	 * file that was distributed with this source code.
+	 */
+
+	_jquery2.default.fn.cmfTree = function (options) {
+	    options = _jquery2.default.extend({
+	        adapter: null,
+	        request: {
+	            load: null
+	        },
+	        actions: {}
+	    }, options);
+
+	    // configure options
+	    var $treeOutput = (0, _jquery2.default)(this);
+	    var selectElement = function selectElement(selector) {
+	        if ('string' == typeof selector) {
+	            return (0, _jquery2.default)(selector);
+	        } else if (selector instanceof _jquery2.default) {
+	            return selector;
+	        }
+
+	        throw 'Cannot handle selector ' + selector + '. You may want to pass a jQuery object or a jQuery selector.';
+	    };
+
+	    if (!options.request.load) {
+	        throw 'cmfTree needs an AJAX URL to lazy load the tree, pass it using the `request.load` option.';
+	    }
+
+	    if (!options.adapter) {
+	        options.adapter = new _fancytree.FancytreeAdapter(options);
+	    }
+	    var adapter = options.adapter;
+
+	    if (!adapter.bindToElement) {
+	        throw 'cmfTree adapters must have a bindToElement() method to specify the output element 
```

Comment:
here is the major change, the aktual key should be added again, its children only.